### ISSUE TITLE
Writing to `stdin.buffer` is using internals which break in 1.8.

### ISSUE THREAD

**Issue by KristofferC** [charleskawczynski/TexasHoldem.jl#106]
Just a heads up that the code at:

https://github.com/charleskawczynski/TexasHoldem.jl/blob/a50dd38acc6728f32573a9230770fda09758f366/test/human_player_option.jl#L12-L19

does not work on the upcoming 1.8:

```
Test fold: Error During Test at /home/pkgeval/.julia/packages/TexasHoldem/RODXB/test/human_player_option.jl:23
  Got exception outside of a @test
  type IOStream has no field buffer
  Stacktrace:
    [1] getproperty(x::IOStream, f::Symbol)
      @ Base ./Base.jl:38
    [2] simulate_keystrokes(::Symbol, ::Vararg{Any})
      @ Main ~/.julia/packages/TexasHoldem/RODXB/test/human_player_option.jl:14
    [3] macro expansion
      @ ~/.julia/packages/TexasHoldem/RODXB/test/human_player_option.jl:26 [inlined]
    [4] macro expansion
      @ /opt/julia/share/julia/stdlib/v1.8/Test/src/Test.jl:1357 [inlined]
    [5] top-level scope
```

Using internal fields like that is always a bit risky in this sense.

PkgEval report: https://s3.amazonaws.com/julialang-reports/nanosoldier/pkgeval/by_hash/f51f24c_vs_2ca8b0c/TexasHoldem.primary.log

**Comment by charleskawczynski** [comment]
Thanks for the heads up! I was trying to fix this and then realized that I got it from Julia's test suite, which seems to still have [something similar]((https://github.com/JuliaLang/julia/blob/8076517c979c703ebed3f1ca5494be2eddfdc30f/stdlib/REPL/test/TerminalMenus/runtests.jl#L7-L22)) (but may be used differently).

**Comment by charleskawczynski** [comment]
There doesn't seem to be any API around testing this without internals. Closed by #108.
